sysutils/jq/Portfile

```diff
@@ -20,11 +21,16 @@ long_description\
 
 homepage            https://jqlang.github.io/jq/
 
-depends_lib         port:oniguruma6
+checksums           rmd160  8931882a8ded5a1d23e42039767ae10c3bf7a349 \
+                    sha256  402a0d6975d946e6f4e484d1a84320414a0ff8eb6cf49d2c11d144d4d344db62 \
+                    size    1905863
 
-checksums           rmd160  7ae650c881fa14ade87569b7e9cd25a62ee6e4e3 \
-                    sha256  5de8c8e29aaa3fb9cc6b47bb27299f271354ebb72514e3accadc7d38b5bbaa72 \
-                    size    1750584
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:libtool
```

Comment:
Doesn't `use_autoreconf yes` already add these ports?

Reply:
Went back and removed this.